metadata-transformer/src/test/java/no/unit/nva/metadata/CreatePublicationRequestTest.java

```diff
@@ -0,0 +1,42 @@
+package no.unit.nva.metadata;
+
+import static no.unit.nva.hamcrest.DoesNotHaveEmptyValues.doesNotHaveEmptyValues;
+import static no.unit.nva.testutils.RandomDataGenerator.randomJson;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static nva.commons.core.attempt.Try.attempt;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import com.fasterxml.jackson.databind.JsonNode;
+import no.unit.nva.commons.json.JsonUtils;
+import no.unit.nva.model.testing.PublicationGenerator;
+import org.junit.jupiter.api.Test;
+
+class CreatePublicationRequestTest {
+
+    @Test
+    void shouldRoundTripToJsonWithoutInformationLoss() {
```

Comment:
Test necessary because CreatePubliatonRequest lies in a different module than where it is used.